ready_ee_pose :
  position :
    x: 0.38692417487303915
    y: 0.179373064137164
    z: 1.1295585061709243
  orientation : # Quaternion
    x: 0.485611001553379
    y: 0.4677052743494253
    z: 0.5195888823680865
    w: 0.5248439052094778

track_duration : 1 # duration for tracking in seconds
max_linear_acceleration : 10 #m/s^2
max_linear_velocity : 1 #m/s
max_angular_acceleration : 17.45 # rad/s^2
max_angular_velocity : 3.14 # rad/s

input_stream_timeout : 0.2 #s
cmd_publish_frequency : 100 #hz

linear_stop_threshold : 0.007 # having a higher threshold means faster stop because of how PID works
angular_stop_threshold : 0.01

linear_track_interpolation_factor : 0.1 # m in the direction of the target from the ready_ee_pose

gains :
  linear : 
    p : 1000
    i : 0
    d : 0
    k : 0.01
  angular : 
    p : 300
    i : 0
    d : 0
    k : 0.01

pre_grasp_transform : # transformation from grasp -> pregrasp
  position :
    x : 0.0
    y : 0.0
    z : -0.15
  orientation :
    w : 1 
    x : 0
    y : 0
    z : 0